graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -303,6 +303,14 @@ def test_graph_nodeProperties_stream_with_arrow(gds: GraphDataScience) -> None:
     y_values = result[result.nodeProperty == "y"]
     assert {e for e in y_values["propertyValue"]} == {2, 3, 4}
 
+    z_values = result[result.nodeProperty == "z"]
+    assert len(z_values) == 3
+    for e in z_values["propertyValue"]:
+        assert e in [[9], [42], [1337]]
```

Comment:
there is no easier assert for python? 

A bit spoiled by assertJ I guess.
But I would hope there is a test util for pandas series